doghouse/server/storage.go

```diff
@@ -0,0 +1,58 @@
+package server
+
+import (
+	"context"
+
+	"google.golang.org/appengine/datastore"
+)
+
+type Installation struct {
+	RepositoryID       int64
+	InstallationID     int
+	RepositoryFullName string
+}
+
+func newInstallationKey(ctx context.Context, repositoryName string) *datastore.Key {
+	kind := "Installation"
+	return datastore.NewKey(ctx, kind, repositoryName, 0, nil)
+}
+
+type CheckSuiteEvent struct {
+	Action     string `json:"action,omitempty"`
+	Repository struct {
+		ID       int64  `json:"id,omitempty"`
+		FullName string `json:"full_name,omitempty"`
+	} `json:"repository,omitempty"`
+	Installation struct {
+		ID int `json:"id,omitempty"`
+	} `json:"installation,omitempty"`
+}
+
+func SaveInstallationFromCheckSuite(ctx context.Context, c CheckSuiteEvent) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function SaveInstallationFromCheckSuite should have comment or be unexported